rootfs-builder/rootfs.sh

```diff
@@ -311,7 +342,16 @@ fi
 
 mkdir -p ${ROOTFS_DIR}
 
-if [ -n "${USE_DOCKER}" ] ; then
+detect_go_version
+
+
+if [ -z "${USE_DOCKER}" ] ; then
+	#Generate an error if the local Go version is too old
+	foundVersion=$(go version | sed -E "s/^.+([0-9]+\.[0-9]+\.[0-9]+).*$/\1/g")
```

Comment:
the `.*$` is just in case the version include a random extra thin like 1.1.1-rc0?

Reply:
I wonder if this should be applied in any case?

Reply:
@jcvenegas when `USE_DOCKER` is set, golang is installed with whatever version set in `GO_VERSION`, so it makes little sense to check the golang version afterwards.

But, it doesn't hurt to check it anyway, if that makes things more linear.

Reply:
@jcvenegas About the `.*$` part, that's because `go version` returns on my machine:
`go version go1.12.5 linux/amd64`
You need to specify a regexp that matches the full line to avoid printing the extra text after `1.12.5`